fix: adjust to `rr`'s vFile reply

RocketMaDev opened the pull request:
`rr replay` use `"F-01,2"` to indicate a vFile error while pwndbg detects `"F-1,"`. Patch the code to process some cases like `rr`.

As this piece of code is run when displaying context, it's critical to fix it. pwndbg tries to cache `/proc/$tid/maps` which utilize vFile to grab it.

RocketMaDev committed:
fix: adjust to `rr`'s vFile reply

`rr replay` use `"F-01,2"` to indicate a vFile error while pwndbg
detects `"F-1,"`. Patch the code to process some cases like `rr`.

* edit: add a ` ` to satisfy lint.

patryk4815 commented on pwndbg/aglib/file.py:
```diff
@@ -170,8 +170,11 @@ def is_vfile_qemu_user_bug() -> bool:
-    if response.startswith(b"F-1,"):
-        errno = int(response[4:].decode(), 10 if is_vfile_qemu_user_bug() else 16)
+    if response.startswith(b"F") and b"," in response:
+        resp_status, resp_data = response[1:].split(b",", 1)  # Skip "F"
```
`resp_data` may return `errno;attachment`, we need `split(';', 1)[0]` here

RocketMaDev replied:
Do you mean `resp_status`?

patryk4815 replied:
`resp_data`

patryk4815 replied:
`‘F result [, errno] [; attachment]’`

RocketMaDev committed:
aglib.file: apply to potential `"errno;attachment"`

* edit: satisfy reviewdog

RocketMaDev committed:
aglib.file: skip `attachment` when parsing vFile

> F result [,errno] [;attachment]

`attachment` is what we don't need, strip it before parsing result and
errno

RocketMaDev committed:
aglib.file: remove duplicated split
